cms/djangoapps/contentstore/views/course.py

```diff
@@ -469,6 +472,56 @@ def _get_rerun_link_for_item(course_key):
     return reverse_course_url('course_rerun_handler', course_key)
 
 
+def _ora1_deprecation_info(course_id, advanced_modules):
```

Comment:
Is there a way to make this a little more generic in the implementation? So that when we deprecate `foldit` or some other type next, it's just a config change and doesn't involve writing a new method?